collector/CMakeLists.txt

```diff
@@ -16,7 +16,13 @@ set(CMAKE_CXX_FLAGS_RELEASE "-O3 -fno-strict-aliasing -DNDEBUG")
 if(ADDRESS_SANITIZER)
 	set(DISABLE_PROFILING ON)
 	set(USE_VALGRIND OFF)
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address -fsanitize=undefined -DGRPC_ASAN_SUPRESSED")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address -fsanitize=undefined -DGRPC_ASAN_SUPPRESSED")
```

Comment:
Did you mean to add this change?

Reply:
Yes, I did, it has been wrong since we first added it.